[playtest 20201213] Lobby server crash

```
[2020-12-27T22:34:34] poop has joined the game.
[2020-12-27T22:35:19] X has joined the game.
[2020-12-27T22:37:44] Game started
[2020-12-27T22:37:44] spetsnaz84 (Spectator) has disconnected.

Unhandled Exception:
System.NullReferenceException: Object reference not set to an instance of an object
  at OpenRA.Mods.Common.Server.LobbySettingsNotification.ClientJoined (OpenRA.Server.Server server, OpenRA.Server.Connection conn) [0x00023] in /tmp/ora-srvwrap/OpenRA-playtest-20201213/OpenRA.Mods.Common/ServerTraits/LobbySettingsNotification.cs:25
  at OpenRA.Server.Server+<>c__DisplayClass33_0.<ValidateClient>b__0 () [0x00236] in /tmp/ora-srvwrap/OpenRA-playtest-20201213/OpenRA.Game/Server/Server.cs:519
  at OpenRA.Server.Server+<>c__DisplayClass33_1.<ValidateClient>b__6 () [0x0021b] in /tmp/ora-srvwrap/OpenRA-playtest-20201213/OpenRA.Game/Server/Server.cs:635
  at OpenRA.Primitives.ActionQueue.PerformActions (System.Int64 currentTime) [0x0006f] in /tmp/ora-srvwrap/OpenRA-playtest-20201213/OpenRA.Game/Primitives/ActionQueue.cs:53
  at OpenRA.Server.Server+<>c__DisplayClass29_0.<.ctor>b__1 (System.Object _) [0x00246] in /tmp/ora-srvwrap/OpenRA-playtest-20201213/OpenRA.Game/Server/Server.cs:301
  at System.Threading.ThreadHelper.ThreadStart_Context (System.Object state) [0x00025] in /build/mono/src/mono/mcs/class/referencesource/mscorlib/system/threading/thread.cs:78
  at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00071] in /build/mono/src/mono/mcs/class/referencesource/mscorlib/system/threading/executioncontext.cs:968
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in /build/mono/src/mono/mcs/class/referencesource/mscorlib/system/threading/executioncontext.cs:910
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state) [0x0002b] in /build/mono/src/mono/mcs/class/referencesource/mscorlib/system/threading/executioncontext.cs:899
  at System.Threading.ThreadHelper.ThreadStart (System.Object obj) [0x0000f] in /build/mono/src/mono/mcs/class/referencesource/mscorlib/system/threading/thread.cs:93
[ERROR] FATAL UNHANDLED EXCEPTION: System.NullReferenceException: Object reference not set to an instance of an object
  at OpenRA.Mods.Common.Server.LobbySettingsNotification.ClientJoined (OpenRA.Server.Server server, OpenRA.Server.Connection conn) [0x00023] in /tmp/ora-srvwrap/OpenRA-playtest-20201213/OpenRA.Mods.Common/ServerTraits/LobbySettingsNotification.cs:25
  at OpenRA.Server.Server+<>c__DisplayClass33_0.<ValidateClient>b__0 () [0x00236] in /tmp/ora-srvwrap/OpenRA-playtest-20201213/OpenRA.Game/Server/Server.cs:519
  at OpenRA.Server.Server+<>c__DisplayClass33_1.<ValidateClient>b__6 () [0x0021b] in /tmp/ora-srvwrap/OpenRA-playtest-20201213/OpenRA.Game/Server/Server.cs:635
  at OpenRA.Primitives.ActionQueue.PerformActions (System.Int64 currentTime) [0x0006f] in /tmp/ora-srvwrap/OpenRA-playtest-20201213/OpenRA.Game/Primitives/ActionQueue.cs:53
  at OpenRA.Server.Server+<>c__DisplayClass29_0.<.ctor>b__1 (System.Object _) [0x00246] in /tmp/ora-srvwrap/OpenRA-playtest-20201213/OpenRA.Game/Server/Server.cs:301
  at System.Threading.ThreadHelper.ThreadStart_Context (System.Object state) [0x00025] in /build/mono/src/mono/mcs/class/referencesource/mscorlib/system/threading/thread.cs:78
  at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00071] in /build/mono/src/mono/mcs/class/referencesource/mscorlib/system/threading/executioncontext.cs:968
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in /build/mono/src/mono/mcs/class/referencesource/mscorlib/system/threading/executioncontext.cs:910
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state) [0x0002b] in /build/mono/src/mono/mcs/class/referencesource/mscorlib/system/threading/executioncontext.cs:899
  at System.Threading.ThreadHelper.ThreadStart (System.Object obj) [0x0000f] in /build/mono/src/mono/mcs/class/referencesource/mscorlib/system/threading/thread.cs:93
```


- OpenRA playtest 20201213
- mono 6.10.0.104
- Linux x86_64

--- a/OpenRA.Mods.Common/ServerTraits/LobbySettingsNotification.cs
+++ b/OpenRA.Mods.Common/ServerTraits/LobbySettingsNotification.cs
@@ -22,9 +22,6 @@ namespace OpenRA.Mods.Common.Server
 		{
 			lock (server.LobbyInfo)
 			{
-				if (server.LobbyInfo.ClientWithIndex(conn.PlayerIndex).IsAdmin)
-					return;
-
 				var defaults = new Session.Global();
 				LobbyCommands.LoadMapSettings(server, defaults, server.Map.Rules);
 

